.github/workflows/playwright.yml

```
@@ -34,25 +34,37 @@ jobs:
+      
       - name: Run Playwright tests
         run: npx playwright test
+      
       - uses: actions/upload-artifact@v4
         if: always()
         with:
           name: playwright-report
           path: playwright-report/
           retention-days: 30
+      
+      - name: Stop server
```

actionlint (1.7.4)

step must run script with "run" section or run action with "uses" section

(syntax-check)